e2e-tests/tests/init-deploy/03-read-from-primary.yaml

```diff
@@ -10,3 +10,5 @@ commands:
       data=$(run_psql_local '\c myapp \\\ SELECT * from myApp;' "postgres:$(get_psql_user_pass init-deploy-pguser-postgres)@$(get_psql_user_host init-deploy-pguser-postgres)")
 
       kubectl create configmap -n "${NAMESPACE}" 03-read-from-primary --from-literal=data="${data}"
+
+      check_passwords_leak
```

Comment:
please add it as a separate step `04-check-pass-leak.yaml`

Reply:
also add it to `monitoring` test in position `06-check-pass-leak.yaml` and to `demand-backup` test into position `08-check-pass-leak.yaml`

Reply:
done